src/benchmarks/micro/libraries/System.Runtime/Perf.Boolean.cs

```diff
@@ -12,8 +12,17 @@ namespace System.Tests
     [BenchmarkCategory(Categories.Libraries)]
     public class Perf_Boolean
     {
-        public static IEnumerable<object> StringValues => Values.Select(value => value.ToString()).ToArray();
-
+        public static IEnumerable<object> StringValues => new string[]{
+            "true",
+            "false",
+            "TRUE",
+            "False"
+        };
+        public static IEnumerable<object> AllStringValues => StringValues.Concat(new string[]
```

Comment:
Nit: consider calling them ValidStringValues and ValidAndInvalidStringValues, or something like that